winwing_mcdu.py

```diff
@@ -146,6 +146,9 @@ class DisplayManager:
             ' ' : 0x0042  # use white
     }
 
+    # alha847, define display background color
+    col_bg = [0x00, 0x00, 0x00]
+
```

Comment:
Why not move it into the init function? It is not needed elsewhere. Don't include usernames in comments. Also I don't think the comment is necessary, variable name explains whats going on.

Reply:
Looks good now. Thank you